src/features/treatments.js

```diff
@@ -116,4 +116,13 @@ export const TREATMENTS_PROD_CSV = `treatment_id,position,view_type,interaction,
 3,7,barchart,none,none,66,6,,110,24,,110,49,1200,300,,,,,,,no,,Barchart MEL question experiment 3 of date/delay paper full screen.
 3,8,barchart,none,none,77,3,,118,16,,120,33,1200,300,,,,,,,no,,Barchart MEL question experiment 3 of date/delay paper full screen.
 3,instructions,barchart,none,none,300,2,,700,7,,1000,15,1200,300,,,,,,,no,introduction-barchart-no-ticks-right,Barchart MEL question experiment 3 of date/delay paper full screen.
+4,1,calendarWord,none,none,350,,1/4/2023,430,,1/13/2023,430,,,,,,,,,,,,Worded MEL question experiment 1 date/delay paper.
```

Comment:
TODO fix the dates here since it should be weeks to match up with MEL word prod treatment